cmd/analyzer/main.go

```diff
@@ -219,6 +230,7 @@ func main() {
 	err = os.WriteFile(outputViolations, b, 0644)
 	if err != nil {
 		log.Error(err, "error writing output file", "file", outputViolations)
+		errLog.Error(err, "error writing output file", "file", outputViolations)
```

Comment:
@eemcmullan is this printing error twice on the console? if yes, why not only keep stderr

Reply:
@pranavgaikwad I asked that we make sure that the error still went to the log file and not just to Stderr. if this is printing twice @eemcmullan I think you probably need to look for a MultiWriter for the file and stdout when a file is set up: https://github.com/Sirupsen/logrus/issues/230#issuecomment-381639138.